app/src/main/res/xml/preferences_playback.xml

```diff
@@ -96,6 +96,12 @@
                 android:key="prefQueueAddToFront"
                 android:summary="@string/pref_queueAddToFront_sum"
                 android:title="@string/pref_queueAddToFront_title"/>
+        <SwitchPreference
+            android:defaultValue="false"
+            android:enabled="false"
+            android:key="prefQueueKeepInProgressAtFront"
+            android:summary="@string/pref_queueKeepInProgressAtFront_sum"
+            android:title="@string/pref_queueKeepInProgressAtFront_title"/>
```

Comment:
Please use the same indention as the rest of the file. I don't think we have an xml style guide yet but most files seem to use 8 spaces for the attributes.

Reply:
will fix it. 
Having said that, in Android Studio, it auto-indented to 4 spaces here. I don't quite know when it indented to 8 and when it indented to 4.